spark-connector/README.md

```diff
@@ -0,0 +1,29 @@
+## Spark Connector examples
+
+Documentation for the KDP Spark Connector can be found at [Koverse.com](https://documentation.koverse.com/)
+
+### Running the examples
+
+To run an example, go through and update the workspaceId, datasetId, etc for your KDP workspace. Run 'mvn package' to build the example. 
+
+You will need to set the environmental variables for your username(email) and password for authentication to KDP.
+
+```
+export KDP_EMAIL=myemail@mycompanydomain.com
+export KDP_PSWD=AbadP@ssw0rd
+```
+
+To run you will need to install Spark (or run the application from your Spark cluster). See [Spark latest documention](https://spark.apache.org/docs/latest/) for installation and instructions on running via spark-submit locally or on a cluster.
+
+Example command:
+```
+$SPARK_HOME/bin/spark-submit ./target/kdp-spark-example.jar --master local[4]
+```
+
+### KDP Count
+
+This example, in Java, covers the main functionality of the KDP Spark Connector. It includes read, write, and simple ABAC. You will need Spark 3, Maven 3, and Java 8 or Java 11 to run this example.
+
+You will first need to add a dataset to your workspace. Go into the src/main/resources directory and upload the file kdp-count-records.json. This will be the datasetId you will start with. You will need to update the variables for workspaceId, datasetId, datasetName to run this example.
+
+Please see [KdpCount.java](spark-connector/kdp-spark-count-example/src/main/java/com/koverse/spark/KdpCount.java) for a walk through of the code.
```

Comment:
When I clicked the link, it took me to this link and not a valid one.

`https://github.com/Koverse/kdp-examples/blob/dd-spark-app-example/spark-connector/spark-connector/kdp-spark-count-example/src/main/java/com/koverse/spark/KdpCount.java`

Reply:
Fixed and verified after merge.